hw9/phone_book_json.py

```diff
@@ -0,0 +1,72 @@
+import json
+
+with open('json_pb.json', 'r+') as file_open:
+    my_phone_book = json.loads(file_open.read())
+
+
+def user_add(user):
+    if user not in my_phone_book.keys():
+        if len(user) > 0:
+            user_number = input(f'pls enter phone number for {user} - ')
+            if len(user_number) > 3:
+                my_phone_book[user] = user_number
+                with open('json_pb.json', 'w') as file_add:
```

Comment:
You will get an error if the file does not exist. Change the modifier to w+ to avoid this problem